src/components/apps/details/AppDoc.js

```diff
@@ -77,13 +76,22 @@ function Documentation(props) {
         baseId,
     } = props;
 
+    const [htmlDocumentation, setHtmlDocumentation] = useState("");
```

Comment:
This starting state is pretty much my main question -- should we have something inforrmative here, or not worry about it?

Reply:
I wouldn't worry too much about it give the doc end-point responds really fast.